.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
> Not as a library your project might depend on.

Not sure to agree. Should php-cs-fixer, gulp, and such tools be considered as applications rather than libraries so? 
As for these packages, there is great chances for that a dotfile will be added to projects using this project

> (BTW the type is currently missing from the composer.json file ^^'

Thank you ☺️